### DISCUSSION TITLE
Make date_before_or_equal and date_after_or_equal

### DISCUSSION THREAD

**Discussion by RomainLanz** [formkit/formkit#1328] [Ideas]
Hey! 👋🏻 

We currently only have rules like `date_before` or `date_after` that strictly enforce that a date is before or after a specific date.
Adding a variant that ensures the same thing but includes the current date would be beneficial.

The current workaround is to take the current date and remove/add one day.
So, when it displays the error message, it is hard for the end user to understand the valid dates.

**Comment by RomainLanz** [comment]
Added!

https://github.com/formkit/formkit/releases/tag/1.6.7

**Reply by justin-schroeder** [reply]
Thanks to you 💪
